lib/protocol/networks.js

```diff
@@ -810,7 +810,7 @@ regtest.type = 'regtest';
 
 regtest.seeds = [];
 
-regtest.magic = 0xfabfb5da;
+regtest.magic = 0xdab5bffa;
```

Comment:
Magic number comes from reversing the order defined in FLO cores [chainparams.cpp](https://github.com/floblockchain/flo/blob/a33e124e82bebd541295d958af23f79efd4774c9/src/chainparams.cpp#L388).